starter_code/src/movies.js

```diff
@@ -1,15 +1,70 @@
 /* eslint no-restricted-globals: 'off' */
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
+function orderByYear(array) {
+    let copyArray = array.slice();
+    return copyArray.sort(function(a, b) {
+      if (a.year > b.year) {
+        return 1;
+      } else {
+        return -1;
+      }
+    });
+  }
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct
+var howManyMovies =  movies.filter(function(dramamovie) {
+	if (dramamovie.genre.includes ("Drama"))
```

Comment:
really nice way of thinking but also you could do && operator it just more cleaner